File: Campaign_Samples/python/bulk_get_campaigns.py
import requests, base64, json
accountSid = "<your Exotel SID>"
accountToken = "<your Exotel token>"
encoding = base64.b64encode(accountSid + ":" + accountToken)

url = "https://api.exotel.com/v2/accounts/"+ accountSid +"/campaigns"

querystring = {"page_size":"1","page":"1"}

payload = ""
headers = {
    'Content-Type': "application/json",
    'Authorization': "Basic " + encoding
    }

response = requests.request("GET", url, data=payload, headers=headers, params=querystring)

print(response.text)

File: Campaign_Samples/js/bulk_get_campaigns.js
var request = require("request");
var accountSid = "<your Exotel Sid>";
var accountToken = "<your Exotel Token>";
var encoding = Buffer.from(accountSid + ':' + accountToken).toString('base64')
var options = { method: 'GET',
  url: 'https://api.exotel.com/v2/accounts/'+ accountSid +'/campaigns',
  qs: { page_size: '1', page: '1' },
  headers: 
   { 
     Authorization: 'Basic ' + encoding,
     'Content-Type': 'application/json' } };

request(options, function (error, response, body) {
  if (error) throw new Error(error);

  console.log(body);
});
